build-tools/packages/build-cli/src/commands/generate/changeset.ts

```diff
@@ -0,0 +1,263 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Package } from "@fluidframework/build-tools";
+import { VersionBumpType } from "@fluid-tools/version-tools";
+import { Flags } from "@oclif/core";
+import chalk from "chalk";
+import humanId from "human-id";
+import { writeFile } from "node:fs/promises";
+import path from "node:path";
+import prompts from "prompts";
+
+import { BaseCommand } from "../../base";
+import { Repository, getDefaultBumpTypeForBranch } from "../../lib";
+
+const DEFAULT_BRANCH = "main";
+const INSTRUCTIONS = `
+↑/↓: Change selection
+Space: Toggle selection
+a: Toggle all
+Enter: Done`;
+
+const excludedScopes = new Set(["@fluid-example", "@fluid-internal", "@fluid-test"]);
```

Comment:
Is this hardcoded option? What about experimental packages?

Reply:
Yes, hardcoded list for now. I didn't exclude experimental packages because we publish them. Any scope that we usually publish should be included by default.